clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -711,9 +718,11 @@ public void handleResponse(AbstractResponse response) {
 
     private class AddPartitionsToTxnHandler extends TxnRequestHandler {
         private final AddPartitionsToTxnRequest.Builder builder;
+        private long retryBackoffMs;
```

Comment:
Worth adding a comment saying that this could be overridden with a pointer to the override method.